lib/commands/serve.js

```diff
@@ -58,6 +58,20 @@ module.exports = Command.extend({
     { name: 'watcher', type: String, default: 'events', aliases: ['w'] },
     { name: 'live-reload', type: Boolean, default: true, aliases: ['lr'] },
     { name: 'live-reload-host', type: String, aliases: ['lrh'], description: 'Defaults to host' },
+    {
+      name: 'live-reload-js-url',
+      type: String,
+      aliases: ['lrju'],
+      description:
+        'The absolute URL used to load livereload.js. If specified, this overrides the liveReloadPort option.',
+    },
+    {
+      name: 'live-reload-options',
+      type: String,
+      aliases: ['lro'],
+      description:
+        'Stringified JSON object for LiveReload options. LiveReload supports a number of options for configuring websocket communication, including https, host, port, and others. See advanced example below.',
```

Comment:
I think the last part of this sentence should be removed ("see below for more info" doesn't seem right). Maybe replace it with a git.io link to the README where it is listed?